src/analyze/CMakeLists.txt

```diff
@@ -21,13 +23,24 @@ target_link_libraries(sharedprint LINK_PUBLIC
 )
 
 if (ZSTD_FOUND)
-target_link_libraries(sharedprint LINK_PUBLIC
-    boost-zstd
+    target_link_libraries(sharedprint LINK_PUBLIC
+        boost-zstd
 )
```

Comment:
if you indent the above, then also indent the line here please

Reply:
Done.